fuzz/fuzz_targets/base.rs

```diff
@@ -8,6 +8,7 @@ fuzz_target!(|data: &[u8]| {
     let mut hpke = Hpke::<hpke_rs_rust_crypto::HpkeRustCrypto>::new(
         HpkeMode::Base,
         KemAlgorithm::DhKemP256,
+        KemAlgorithm::DhKemK256,
```

Comment:
You can run the fuzzer with `cargo +nightly fuzz run base` and will see the compilation error. This line should go.